tests/meta.js

```diff
@@ -6,20 +6,19 @@ import process from 'child_process';
 import Promise from 'bluebird';
 
 test('get packages', t => {
```

Comment:
That's an Istanbul error, don't know why it's freaking out. Mine too. Don't know why it's trying to do an HTML report. I think it's an NYC issue. I'll file it with them